dhcpv4/nclient4/client.go

```diff
@@ -492,7 +492,7 @@ func (e *ErrNak) Error() string {
 	return "server rejected request with Nak"
 }
 
-// RequestFromOffer sends a Request message and waits for an response.
+// RequestFromOffer sends a Request message in the SELECT state and waits for a response.
```

Comment:
Same: Do we need this clarification? I am not sure if this makes things more clear or more confusing.

Reply:
This calls `NewRequestFromOffer`, which does SELECT requests by default. I can however change the wording to be a bit more subtle.